wiki/Guides/Compressing_files/en.md

```diff
@@ -29,27 +29,49 @@ This section will show you how to remove audio from videos using [Handbrake](htt
 
 ![Removing audio tracks from the audio tab in Handbrake](img/removeaudio-handbrake.jpg "Removing the audio tracks")
 
-4. Go into the `Video` tab and set the video codec as `H.264 (X264)`. Change the `Constant Quality` to between 20–25. Smaller values will produce smaller file sizes at the cost of the video's quality.
+4. Go into the `Video` tab and make sure the video codec is set as `H.264 (X264)`. Change the `Constant Quality` to between 20–25. Smaller values will produce larger file sizes but with a higher video quality.
+
+5. If you're willing to spend more time encoding slower Encoder Presets deliver better video quality and may also reduce the video file size, but do not go down to placebo as it is prohibitively slow and often performs worse than VerySlow.
 
 ![Setting the video codec and quality in Handbrake](img/codecquality-handbrake.jpg "Setting the video codec and constant quality")
 
-5. Set the framerate to `30`.
+6. Set the framerate to be the same as the source and set it to constant framerate.
 
 ![Setting the video framerate in Handbrake](img/framerate-handbrake.jpg "Setting the framerate")
 
-6. To resize the image of the video file, go to the `Dimensions` tab and change the width to `1280` and change the height to `720`.
+7. To resize the image of the video file, go to the `Dimensions` tab and change the width to `1280` and change the height to `720`.
```

Comment:
Does this only downscale?

Reply:
apparently not (anymore?). seems like these images are very outdated

![image](https://user-images.githubusercontent.com/36758269/135710860-51e49445-9281-471f-b886-f975d80a5f13.png)

Reply:
lol I still have the old UI for this... *same as the old image and not like above*

ig I would need to redo some of the screenshots and update the handbrake that I have locally.

Reply:
Looks like this new UI was released as part of the handbrake 1.4 update (dates back to July 2021).

ig it might be a good idea to pop in a notice to check for updates if the ui looks different?